index.html

```diff
@@ -2314,6 +2341,15 @@ <h4>Interaction URL Format</h4>
 [[[#interaction-protocols-response]]], into the query parameters of the
 interaction URL.
         </p>
+
+        <p class="note" title="Interactions URL lives on coordinators, not Workflow Services">
+A coordinator has the freedom to put the interaction URL wherever it wants as it is expected 
+to be consumed by the participating party via some other transport layer (QR Code, Bluetooth, etc), 
+but needs to format the data returned per this specification. To be clear, the interaction URL is 
+expected to be hosted on a coordinator, as business rules may modulate actions, and NOT hosted at 
+workflow service. Someone consuming the interaction URL should treat the URL as opaque, and key off 
+of the “iuv=1” query parameter to recognise the interaction URL. 
```

Comment:
This line (2351, as of this writing) should be deleted, if [my suggestion above](https://github.com/w3c-ccg/vcalm/pull/563/files#r2446245808) is merged

```suggestion
```